tests/test_icml_conference.py

```diff
@@ -5435,3 +5367,160 @@ def test_forum_chat(self, openreview_client, helpers):
+        assert len(openreview_client.get_messages(to='sac2@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 0
+        assert len(openreview_client.get_messages(to='pc@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 0
+
+        note_edit = sac_client.post_note_edit(
+            invitation='ICML.cc/2023/Conference/Submission1/-/Chat',
+            signatures=['ICML.cc/2023/Conference/Submission1/Senior_Area_Chairs'],
+            note=openreview.api.Note(
+                replyto=note_edit['note']['id'],
+                content={
+                    'message': { 'value': 'Chat comment number 6' }
+                }
+            )
+        )
+
+        helpers.await_queue_edit(openreview_client, edit_id=note_edit['id'])
+
+        assert len(openreview_client.get_messages(to='reviewer1@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='reviewer2@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='melisa@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='reviewer3@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='reviewer4@yahoo.com', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='rachel_bis@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='ac2@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+        assert len(openreview_client.get_messages(to='sac2@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 0
+        assert len(openreview_client.get_messages(to='pc@icml.cc', subject='[ICML 2023] New messages in committee members chat for submission 1: Paper title 1 Version 2')) == 1
+
+        ## Add tag emoji
+        tag = sac_client.post_tag(openreview.api.Tag(
+            invitation='ICML.cc/2023/Conference/Submission1/-/Chat_Reaction',
+            signatures=['ICML.cc/2023/Conference/Submission1/Senior_Area_Chairs'],
+            tag='😄',
+            replyto=note_edit['note']['id']
```

Comment:
Can we change this tag to reply to one of the chat notes? Right now it's a tag on the forum note.

Reply:
`note_edit['note']['id']` is pointing to the previous chat comment "Chat comment number 6", the forum note should be the same for all of them. What am I missing?